src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -277,7 +281,17 @@ public function setFooterTitle(?string $title): static
      */
     public function setHorizontal(bool $horizontal = true): static
     {
-        $this->horizontal = $horizontal;
```

Comment:
The variable `$horizontal` can be removed. It's private and not used.

Reply:
Thanks